docs/source/documentation/concepts.rst

```diff
@@ -177,3 +177,39 @@ There are two ways to run several tasks in parallel on a same organization. The
 .. TODO:: Detail vertical and horizontal scaling
 
 .. TODO:: Explain what is substra tools
+
+
+Compute plan execution - deployed mode
+---------------------------------------
+
+Read this paragraph to understand what happens during the compute plan execution in deployed mode and what can be done to improve the execution time.
+In local mode, these steps are either skipped or simplified.
+
+Once a compute plan is submitted to the platform, its tasks are scheduled to be executed on each organization.
+
+On each organization, Substra fetches the assets needed for the first task, builds the Docker image of the algo and creates a container with the relevant assets. The task executes and Substra saves its outputs.
+Afterwards, every task **from the same compute plan** that uses the same Algo is executed in the same container.
+
+Asset preparation
+^^^^^^^^^^^^^^^^^^
+
+The first step of the task execution is to fetch the necessary assets.
+These include the inputs (e.g. the algo or opener files), the output of other tasks (input artifacts of the task) and data samples.
+
+The assets, data samples excluded, come from the file systems of the organizations. If they are stored on other organizations, they are downloaded over HTTPS connections.
+Example: an algo submitted on another organization.
+
+The data samples are stored on the organization, in a storage solution (MiniO). They are downloaded for the task, this may take a long time if the dataset is large.
```

Comment:
> The data samples are stored **on the organization**

is this enough? How would you say it?

Reply:
Ok to say things differently, for a user without knowledge about how substra works, **download** will imply going outside or inside an organisation.

```suggestion
The data samples are stored on the organization, in a storage solution (MiniO). They are downloaded from the organization storage solution (Mini0) to the organization filesystem? task worskspace?), this may take a long time if the dataset is large.
Note that data samples never leave the organization.
```

Reply:
thank you for the suggestion, I changed the sentence, is it better?